Parser/pegen/pegen.c

```diff
@@ -178,12 +178,38 @@ tokenizer_error(Parser *p)
         case E_EOFS:
         case E_EOLS:
             return tokenizer_string_error(p);
+        case E_INTR:
```

Comment:
One more case needs to be added here: `E_IDENTIFIER`.

Reply:
Isn't `E_IDENTIFIER` already handled in line 174?

Reply:
Oops. I forgot we had already done that. Sorry!